Contest_Archive_Format.md

```diff
@@ -138,17 +140,19 @@ data.
 
 ### Design principles
 
-Endpoints are stored in a single <endpoint>.json file containing the
+Endpoints are stored in a single \<endpoint>.json file containing the
 full list of objects. This file is identical to the API call to
-/<endpoint>. If there are file references in the JSON file these are
+/\<endpoint>. If there are file references in the JSON file these are
```

Comment:
```suggestion
`/<endpoint>`. If there are file references in the JSON file these are
```

Wouldn't is be nicer (apart from the effort) to make all endpoints be `code`, so that the `<>` render without issues?

Reply:
Might be a good idea. Should I do this?